packages/components/CHANGELOG.md

```diff
@@ -1,8 +1,12 @@
-# Unreleased
+# 7.1.0
 
--   Revert Card component removal #7167.
--   Filters: On update respect all other queries, not just persistedQueries #7155
 -   Add rowKey prop to Table and TableCard component. #7196
+-   Filters: On update respect all other queries, not just persistedQueries #7155
+-   Fix non-string query prop warning in SelectControl component. #7046
+-   Fix WordPress 5.8 compatibility UI fixes #7255
+-   Revert Card component removal #7167.
+-   Update DynamicForm, adding initial config memoization. #7256
+-   Update package dependencies
```

Comment:
Was this spread across too many PRs to list?  I know #7010 and #7202.  Not sure if there were others.

Reply:
Yeah I think just two PRs, same for the data package. Given the PR updates where mostly un-related aside from the dependency updates, I deemed it not necessary. But I could add them in if you think that be helpful?

Reply:
It's up to you.  I have no strong opinion one way or another; just caught my eye scanning this PR.